skfda/tests/test_classifier_classes.py

```diff
@@ -0,0 +1,59 @@
+"""Tests classes attribute of classifiers."""
+
+import unittest
+from typing import List
+
+import numpy as np
+
+from skfda._utils._sklearn_adapter import ClassifierMixin
+from skfda.datasets import make_multimodal_samples
+from skfda.ml.classification import LogisticRegression
+from skfda.representation import FData
+
+from ..typing._numpy import NDArrayAny
+
+
+class TestClassifierClasses(unittest.TestCase):
+    """Test for classifiers classes."""
+
+    def setUp(self) -> None:
+        """Establish train and test data sets."""
+        # List of classes to test
+        # Adding new classes to this list will test the classifiers with them
+        self.classes_list: List[NDArrayAny] = [
+            np.array([0, 1]),
+            np.array(["class_a", "class_b"]),
+        ]
+
+        # Create one target y data of length n_samples from each class set
+        n_samples = 30
+        self.y_list = [
+            np.resize(classes, n_samples)
+            for classes in self.classes_list
+        ]
+        self.X = make_multimodal_samples(
```

Comment:
I would rather use `make_gaussian process` unless you really need something that only `make_multimodal_samples` could provide. Gaussian processes are the equivalent to the normal distribution for functional data. The multimodal samples are used only in registration examples.